tests/scripts/task_python_unittest.sh

```diff
@@ -19,10 +19,17 @@
 set -e
 set -u
 
-export PYTHONPATH=python:topi/python
```

Comment:
is this change necessary? it would be good to keep this file untouched, since it will burden the merge task in the future

Reply:
I agree. But this will fail CI due to i386 using 32-bit Python. Is it possible to update it?